js/graphData.js

```diff
@@ -1,181 +1,149 @@
 var chart = c3.generate({
-  bindto: "#scatterChart",
+  bindto: '#scatterChart',
   size: {
-    height: 500
+    height: 500,
   },
   data: {
     // type:'spline',
-    x: "x",
+    x: 'x',
     xFormat: '%Y/%m/%d %H:%M:%S',
     // xFormat: '%Y/%m/%d',
-    columns:[
-      ["x",0],//,'2020/8/18 1:35:45','2020/8/18 1:41:45']
-
-
-      ["Soil Temp (°C)", 0],
-      ["Soil Moisture (%)", 0],
-      ["Air Temp (°C)", 0],
-      ["Humidity (%)", 0],
+    columns: [
+      ['x', 0], //,'2020/8/18 1:35:45','2020/8/18 1:41:45']
+      ['Soil Temp (°C)', 0],
+      ['Soil Moisture (%)', 0],
+      ['Air Temp (°C)', 0],
+      ['Humidity (%)', 0],
       ['Heat Index (°C)', 0],
-      ["Sunlight (%)", 0],
-      ["Watered Today (0/1)", 0]
+      ['Sunlight (%)', 0],
+      ['Watered Today (0/1)', 0],
     ],
     axes: {
-      "Watered Today (0/1)": 'y2' // Add second y-axis for watered_today
-    }
+      'Watered Today (0/1)': 'y2', // Add second y-axis for watered_today
+    },
   },
 
   // Axes formatting
   axis: {
     // x axis
     x: {
-      type:'timeseries',
+      type: 'timeseries',
       tick: {
         format: '%Y-%m-%d %H:%M:%S',
-        rotate: 45
+        rotate: 45,
       },
       label: {
         text: 'Date',
-        position: 'center'
-      }
+        position: 'center',
+      },
     },
 
     // left-hand y axis
     y: {
       label: {
         text: 'Temperature (°C), Percentage (%)',
-        position: 'outer-middle'
-      }
+        position: 'outer-middle',
+      },
     },
 
     // right-hand y axis
     y2: {
       show: true, // add second y-axis
       label: {
-        text: "Watered Today (0 = No, 1 = Yes)",
-        position: 'outer-middle'
+        text: 'Watered Today (0 = No, 1 = Yes)',
+        position: 'outer-middle',
       },
       tick: {
         count: 2, // We don't need float tick values between a "yes" and "no" option
-        values: [0,1] // makes sure values aren't floating points
+        values: [0, 1], // makes sure values aren't floating points
       },
-    }
-  }
+    },
+  },
 });
 
-
-
 // Get data from database, parse into arrays
 async function getJSONData() {
-    var unix_time = [];
-    var date = [];
-    var soil_temp = [];
-    var soil_moisture = [];
-    var air_temp = [];
-    var air_humidity = [];
-    var heat_index = [];
-    var sunlight = [];
-    var watered_today = [];
-    
-    
-    date.push("x");
-    soil_temp.push("Soil Temp (°C)");
-    soil_moisture.push("Soil Moisture (%)");
-    air_temp.push("Air Temp (°C)");
-    air_humidity.push("Humidity (%)");
-    heat_index.push("Heat Index (°C)");
-    sunlight.push("Sunlight (%)");
-    watered_today.push("Watered Today (0/1)");
-    
-    
-    try {
-        $.get("../databaseconnect.php",
```

Comment:
You were pretty close to getting this to work. So $.get() is asynchronous meaning the programming continues to run while this function makes a network request. So the return statement below was executing before the data was retrieved from your php server.